src/analysis/properties.rs

```diff
@@ -200,9 +203,21 @@ fn analyze_property(
         if let Ok(ref s) = used_rust_type(env, prop.typ, true) {
             imports.add_used_type(s);
         }
+        let set_bound = PropertyBound::get(env, prop.typ);
         if type_string.is_ok() {
             imports.add("gobject_sys");
             imports.add("glib::Value");
+            if set_bound.is_some() {
+                imports.add("glib::object::IsA");
+                imports.add("glib::value::SetValueOptional");
+                if !*nullable {
+                    //TODO: support nonnulable setter if found any
```

Comment:
No nonnulable setters in gtk-rs

Reply:
What does this mean? :)

Reply:
I thought that maybe exists some property that can't be NULL,
then generator support this case too (with `param: &T` instead of `param: Option<&T>`).
But check shows that no that properties in gtk-rs crates, so I just add warning and ToDo.

Reply:
Sounds fine to me